### ISSUE TITLE
[COGNITION]: STEM Automated Decision - Consider adding an aria-describedby to the inside/outside US parentheticals

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#22599]
# [508-defect-4](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-4)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Trevor_

## User Story or Problem Statement

As a screen reader user, I would like to know the correct number to call whether inside the US or outside the US. We have this text on the screen, but it's not semantically associated with the links. We could add an `aria-describedby` to remedy this.

## Details

The paragraph block will need a span with a unique ID and the link will need an aria-describedby attribute. Screenshot and code snippet below.

## Acceptance Criteria

- [ ] 0 axe browser plugin and/or CI violations reported
- [ ] Screen readers announce the link, then "inside the US" or "outside the US" after a short pause

## Proposed Solution (if known)

```diff
<p>
  <a
+   aria-describedby="inside-US-tele"
    class="no-wrap"
    href="tel:+18884424551"
    aria-label="8 8 8. 4 4 2. 4 5 5 1."
>888-442-4551</a>&nbsp;
- (inside the U.S.)
+ <span id="inside-US-tele">(inside the U.S.)</span>
</p>
```

## WCAG or Vendor Guidance (optional)

* [Name, Role, Value: Understanding SC 4.1.2](https://www.w3.org/TR/UNDERSTANDING-WCAG20/ensure-compat-rsv.html)

## Screenshots or Trace Logs (optional)
![Screen Shot 2021-04-02 at 3 10 57 PM](https://user-images.githubusercontent.com/934879/113451467-a622b700-93c7-11eb-88e0-2bf9ccabbc73.png)

**Comment by ksantiagoBAH** [comment]
Pull Request: https://github.com/department-of-veterans-affairs/vets-website/pull/16790

**Comment by delli-gatti-michael** [comment]
@1Copenut This is on Staging and ready for your review

**Comment by 1Copenut** [comment]
@delli-gatti-michael I tested this in VoiceOver and JAWS today. When tabbing, it doesn't read the `aria-describedby` as expected when I focus the two links. Same when I move through the page by arrow keys. This will need the Telephone component updated to accept additional props like ID and aria-describedby, as Mottie referenced in this comment: https://github.com/department-of-veterans-affairs/vets-website/pull/16790/files#r614831144

**Comment by ksantiagoBAH** [comment]
@copenut @Mottie Was the `<Telephone/>` component ever updated to accept attributes like an `id` and `aria-describedby`? https://github.com/department-of-veterans-affairs/vets-website/pull/16790/files#r614831144

**Comment by Mottie** [comment]
@ksantiagoBAH It hasn't been - https://design.va.gov/storybook/?path=/docs/components-telephone--default
